.github/workflows/ubuntu-cminpack-install.yaml

```diff
@@ -12,20 +12,15 @@ jobs:
         BUILD_SHARED_LIBS: [ 'ON' , 'OFF' ]
         CMAKE_BUILD_TYPE: [ 'Release', 'Debug', 'RelWithDebInfo', 'MinSizeRel' ]
         CMINPACK_PRECISION: [ 'd' ] # seems to be the only precision that the current tests reference work
-        USE_BLAS: [ 'OFF' ]
-        include:
-          - BUILD_SHARED_LIBS: 'ON'
-            CMAKE_BUILD_TYPE: 'RelWithDebInfo'
-            CMINPACK_PRECISION: 'd'
-            USE_BLAS: 'ON'
+        USE_BLAS: [ 'ON', 'OFF' ]
     
     env:
       CMINPACK_BUILD_DIR: $RUNNER_TEMP/cminpack-build-shared-libs-${{ matrix.BUILD_SHARED_LIBS }}-build-type-${{ matrix.CMAKE_BUILD_TYPE }}-precision-${{ matrix.CMINPACK_PRECISION }}
       CMINPACK_INSTALL_DIR: $RUNNER_TEMP/cminpack-install-shared-libs-${{ matrix.BUILD_SHARED_LIBS }}-build-type-${{ matrix.CMAKE_BUILD_TYPE }}-precision-${{ matrix.CMINPACK_PRECISION }}
     
     steps:
       - name: Install BLAS
-        run: sudo apt-get install -y libopenblas-dev
```

Comment:
what is wrong with openblas? most - if not all - users would prefer openblas over the Fortran BLAS. Fortran BLAS is not optimized for modern CPU architectures